src/bucket/BucketListSnapshot.cpp

```diff
@@ -75,22 +75,28 @@ SearchableBucketListSnapshot::getLedgerEntry(LedgerKey const& k)
 
     if (threadIsMain())
     {
-        auto timer = mSnapshotManager.getPointLoadTimer(k.type()).TimeScope();
-        return getLedgerEntryInternal(k);
+        mSnapshotManager.startPointLoadTimer();
+        auto [result, bloomMiss] = getLedgerEntryInternal(k);
+        mSnapshotManager.endPointLoadTimer(k.type(), bloomMiss);
+        return result;
     }
     else
     {
-        return getLedgerEntryInternal(k);
+        auto [result, bloomMiss] = getLedgerEntryInternal(k);
+        return result;
     }
 }
 
-std::shared_ptr<LedgerEntry>
+std::pair<std::shared_ptr<LedgerEntry>, bool>
 SearchableBucketListSnapshot::getLedgerEntryInternal(LedgerKey const& k)
 {
     std::shared_ptr<LedgerEntry> result{};
+    auto sawBloomMiss = false;
 
     auto f = [&](BucketSnapshot const& b) {
-        auto be = b.getBucketEntry(k);
+        auto [be, bloomMiss] = b.getBucketEntry(k);
```

Comment:
So I think we definitely should not count bloom misses in our normal point load metrics, as this causes too much noise on the extremes. I'm not sure what the best way to track bloom miss metrics is though, if at all. I agree that having one separate timer for all bloom misses may not be very helpful, especially that there are a mix of types, sizes, potential for multiple bloom misses, etc. It didn't feel right not recording bloom miss loads at all, and they are rare enough that it would not be useful to track double misses or per type, so I just threw them all in one metric. If this metric doesn't make sense than we can throw it out and just meter the "happy path" (since we have a bloom miss percentage metric already that will tell us if bloom filters are behaving incorrectly), but I think we definitely should only track bloom hits in the regular metrics.

Reply:
> If this metric doesn't make sense then we can throw it out and just meter the "happy path" (since we have a bloom miss percentage metric already that will tell us if bloom filters are behaving incorrectly), but I think we definitely should only track bloom hits in the regular metrics.

I think that makes sense.